scripts/includes/common.sh

```diff
@@ -1,3 +1,9 @@
+function get_home() {
+  USER=$1
+  eval echo "~$USER";
```

Comment:
Also after some thinking, do we need this function to be flexible and rely on user param?

I mean we always rely on `stanley` in default installer and with that we can just simplify this to:
```diff
- eval echo "~$USER"
+ echo ~stanley
```

and get rid of nasty `eval`

Reply:
If we want to always rely on stanley in default installer, then we can simplify this. With this in mind, I'll make the necessary changes.